src/zjs_callbacks.c

```diff
@@ -95,6 +95,14 @@ static int zjs_ringbuf_error_count = 0;
 static int zjs_ringbuf_error_max = 0;
 static int zjs_ringbuf_last_error = 0;
 
+#ifndef ZJS_LINUX_BUILD
```

Comment:
This can apply to some other modules as well, would it be better if this is in zjs_utils?